SPEC.md

```
@@ -0,0 +1,121 @@
+---
+last-synced-to-vision: 2026-04-02
+status: ACTIVE
+---
+
+# SPEC
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)